### PR TITLE
Updated - DX-130, 134

### PR DESCRIPTION
**Pull request by sushmangupta**

### CODE REVIEW

File: src/apps/administration/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Build
+
+The [Administration module](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains infrastructure and assets defined and used in the Administration user interface.
+
+Shopware server builds the [B2B-Suite Administration](https://developer.shopware.com/docs/products/extensions/b2b-suite/guides/administration). However, you can modify the appearance and extend capabilities of the Administration by customizing its components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/administration/writing-templates), [mixins](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-mixins), [fields](https://developer.shopware.com/docs/guides/plugins/plugins/administration/using-custom-fields), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-filter), [modules](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-module), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-styles), etc.
```
Comment (elkmod) [inline src/apps/administration/index.md:11]:
```suggestion
You can modify the appearance and extend capabilities of the Administration by customizing its components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/administration/writing-templates), [mixins](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-mixins), [fields](https://developer.shopware.com/docs/guides/plugins/plugins/administration/using-custom-fields), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-filter), [modules](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-module), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-styles), etc.
```

File: src/apps/administration/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This can be done via [apps](https://developer.shopware.com/docs/guides/plugins/apps/administration) or [plugins](https://developer.shopware.com/docs/guides/plugins/plugins/administration).
+
+### Rules
```
Comment (elkmod) [inline src/apps/administration/index.md:15]: Not related to the administration

File: src/apps/administration/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### Flow Builder
+
+[Flow Builder](https://developer.shopware.com/docs/guides/plugins/plugins/framework/flow/add-flow-builder-action#add-custom-action-in-administration) lets you build applications to automate a business process.
+
+### CMS
+
+Once the Administration component is built, the representation is handled by [CMS](https://developer.shopware.com/docs/guides/plugins/plugins/content/cms)
+
+### Admin API
```
Comment (elkmod) [inline src/apps/administration/index.md:27]: Not related to administration.

File: src/apps/administration/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Administration Overview
+
+The Administration is the merchant interface of every Shopware store.
+
+It comprises UI for business logic, purchasing logic, data, and capabilities. It outlines all internal processes like product inventory tracking, order processing, and customer record management and gets input from frontend applications.
+
+## Build
+
+The [Administration module](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains infrastructure and assets defined and used in the Administration user interface.
```
Comment (Christian-Rades) [inline src/apps/administration/index.md:9]: I'd be careful with using the term Infrastructure in this context, considering we have 3 different hosting approaches.

File: src/apps/administration/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+You can modify the appearance and extend capabilities of the Administration by customizing its components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/administration/writing-templates), [mixins](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-mixins), [fields](https://developer.shopware.com/docs/guides/plugins/plugins/administration/using-custom-fields), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-filter), [modules](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-module), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-styles), etc.
+
+This can be done via [apps](https://developer.shopware.com/docs/guides/plugins/apps/administration) or [plugins](https://developer.shopware.com/docs/guides/plugins/plugins/administration).
```
Comment (Christian-Rades) [inline src/apps/administration/index.md:13]: The things apps and plugins can do in this area currently differ somewhat. I'm fine with leaving it as is and having the linked pages specify this, but splitting the overview might also make sense :thinking:

File: src/apps/administration/index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+You can create [custom rules](https://developer.shopware.com/docs/guides/plugins/plugins/framework/rule/add-custom-rules) to handle almost anything ecommerce-related for the customer, product or cart. It gives you control over what you want to happen if a specific condition is fulfilled.
+
+### Flow Builder
+
+[Flow Builder](https://developer.shopware.com/docs/guides/plugins/plugins/framework/flow/add-flow-builder-action#add-custom-action-in-administration) lets you build applications to automate a business process.
+
+### CMS
+
+Once the Administration component is built, the representation is handled by [CMS](https://developer.shopware.com/docs/guides/plugins/plugins/content/cms)
```
Comment (Christian-Rades) [inline src/apps/administration/index.md:25]: This sounds like the cms handles the representation of the Administration. I'd describe it more as a management UI for the storefront content.

File: src/apps/administration/index.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+Building the Administration component also involves creating necessary API interfaces and services. 
+
+To provide CRUD operations for every entity within Shopware, the [Admin API](https://developer.shopware.com/docs/concepts/api/admin-api) offers a full range of commerce options making it possible for an ecommerce business to handle backend data.
+
+Coding standards, like for many large projects, Shopware Open Source also has [coding standards](https://developer.shopware.com/docs/resources/guidelines/code). Use these standards when you contribute to the codebase or create extensions.
+
+## Compile
+
+Administration loads configurations from the database and compiles them. To allow compiling without a database, refer to this [guide](https://developer.shopware.com/docs/guides/hosting/installation-updates/deployments/build-w-o-db#compiling-the-administration-without-database)
```
Comment (Christian-Rades) [inline src/apps/administration/index.md:37]: The compilation is more of a step towards a working Administration.

**Review by Christian-Rades — APPROVED** [review]

File: src/apps/administration/index.md (hunk 7 above)
Comment (sushmangupta) [reply src/apps/administration/index.md:37]: Removed!

File: src/apps/administration/index.md (hunk 5 above)
Comment (sushmangupta) [reply src/apps/administration/index.md:13]: May I know what you meant by "splitting the overview"? 🤔

File: src/apps/administration/index.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Administration Overview
+
+The Administration is the merchant interface of every Shopware store. It comprises UI for business logic, purchasing logic, data, and capabilities. It outlines all internal processes like product inventory tracking, order processing, and customer record management and gets input from frontend applications.
+
+The [Administration module](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains framework, component and assets defined and used in the Administration user interface.
```
Comment (Christian-Rades) [inline src/apps/administration/index.md:5]:
```suggestion
The [Administration](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains framework, component and assets defined and used in the Administration user interface.
```

A module is a sub-menu of the administration, I wouldn't mix it with the word Administration.

File: src/apps/administration/index.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+You can create [custom rules](https://developer.shopware.com/docs/guides/plugins/plugins/framework/rule/add-custom-rules) to handle almost anything ecommerce-related for the customer, product or cart. It gives you control over what has to happen if a specific condition is fulfilled.
+
+### Flow Builder
+
+[Flow Builder](https://developer.shopware.com/docs/guides/plugins/plugins/framework/flow/add-flow-builder-action#add-custom-action-in-administration) lets you build workflow to automate a business process.
+
+### Admin API
+
+Building the Administration component also involves creating necessary API interfaces and services.
```
Comment (Christian-Rades) [inline src/apps/administration/index.md:21]: Maybe `extending` is a better word here than building?

**Review by Christian-Rades — CHANGES_REQUESTED** [review]

**Pull request merged**